libc/src/math/generic/log1p.cpp

```diff
@@ -975,16 +975,18 @@ LLVM_LIBC_FUNCTION(double, log1p, (double x)) {
   double err_hi = ERR_HI[hi == 0.0];
 
   // Scaling factior = 2^(-xh_bits.get_exponent())
```

Comment:
since you flipped the subtraction in the calculation of `s_u` is this comment still accurate?

Reply:
yes, it's still the same scaling factor, but we do subtracting exponents (ldexp style) instead of multiply by `2^(-exp)`.  I updated the comment to make it a bit clearer.